resdk/tests/functional/analysis/e2e_expressions.py

```diff
@@ -14,37 +14,37 @@ def test_cuffquant(self):
         bam_1, bam_2 = self.get_bams(2, collection)
         # pylint: enable=unbalanced-tuple-unpacking
         bam_3 = self.get_bams(1)[0]
-        gff = self.get_gtf(1)
+        gff = self.get_gtf(collection)
```

Comment:
You can also leave this empty as annotation does not have to be in a collection.